src/sql/azdata.proposed.d.ts

```diff
@@ -399,5 +400,43 @@ declare module 'azdata' {
 	export interface TaskInfo {
 		targetLocation?: string;
 	}
+	// Assessment interfaces
+	export enum SqlAssessmentTargetType {
```

Comment:
And actually - could you put these in their own namespace so we aren't cluttering up the root namespace so much?